hatch/src/generators/tup/builder.rs

```diff
@@ -1,15 +1,15 @@
 use constants::CATCH_HEADER_URL;
 use generators::platform_kind::PlatformKind;
 use generators::project_asset::ProjectAsset;
-use generators::tup::catch_definition::make_catch_definition_string;
-use generators::tup::catch_header::catch_header_filename;
 use generators::tup::platform::{linux, mac_os, windows};
-use generators::tup::test_tupfile::make_test_tupfile_string;
-use generators::tup::tup_config::make_tup_config_string;
 use generators::tup::tup_kind::TupKind;
-use generators::tup::tupfile::make_tupfile_string;
-use generators::tup::tupfile_ini::TupfileIni;
-use generators::tup::tuprules::make_tuprules_string;
+use generators::tup::catch_definition;
+use generators::tup::catch_header;
+use generators::tup::test_tupfile;
+use generators::tup::tup_config;
+use generators::tup::tupfile;
+use generators::tup::tupfile_ini;
+use generators::tup::tuprules;
```

Comment:
do we want to squash these `use` statements into the one line syntax? e.g. 
```
use generators::tup::{ catch_definition, catch_header, test_tupfile, tup_config, tupfile, tupfile_ini, tuprules };
```

alternatively, we may bve using the whole module. It might be ok importing everything. Rust optimizes these imports anyways:

```
use generators::tup::*
```